peps/pep-0815.rst

```diff
@@ -24,7 +24,7 @@ Abstract
 The Python wheel packaging format uses
 :doc:`packaging:specifications/platform-compatibility-tags` to specify
 a given wheel’s supported environments. These tags are unable to express
-features of modern hardware configuration, such as the availability of
+the features of modern hardware configurations, such as the availability of
```

Comment:
```suggestion
features of modern hardware configurations, such as the availability of
```

We're referring only to specific features that modern hardware has, not all of them.